packages/contracts/src/Admin.sol

```diff
@@ -20,8 +22,7 @@ contract Admin is IMembership, PluginCloneable, ProposalUpgradeable {
     using SafeCastUpgradeable for uint256;
 
     /// @notice The [ERC-165](https://eips.ethereum.org/EIPS/eip-165) interface ID of the contract.
-    bytes4 internal constant ADMIN_INTERFACE_ID =
-        this.initialize.selector ^ this.executeProposal.selector;
+    bytes4 internal constant ADMIN_INTERFACE_ID = this.executeProposal.selector;
```

Comment:
Same as for multisig
https://github.com/aragon/multisig-plugin/pull/19#pullrequestreview-2354876183

This interface is now deprecated